### PR TITLE
[KIALI-242] Add Jaeger Info to Backend

### PR DESCRIPTION
**Pull request by aljesusg**
Provide the url for jaeger

jaeger_url

@jotak I moved some methods of Grafana (Get service...) to utils because I need them to so please review this because is so similar to your code.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -36,6 +36,11 @@ const (
 	EnvGrafanaVarServiceSource = "GRAFANA_VAR_SERVICE_SOURCE"
 	EnvGrafanaVarServiceDest   = "GRAFANA_VAR_SERVICE_DEST"
 
+	EnvJaegerDisplayLink      = "JAEGER_DISPLAY_LINK"
```
Comment (jotak) [inline config/config.go]: Apparently we won't want to turn off jaeger link, as we want to fully leverage it, unlike Grafana. So I think this setting is not necessary.

Comment (aljesusg) [reply config/config.go]: Ok I am gonna remove it

**Comment by aljesusg** [comment]
Added @jotak. Thanks

**Comment by aljesusg** [comment]
Thanks @jotak

**Pull request merged**